front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/StrategyController.groovy

```diff
@@ -48,6 +48,15 @@ class StrategyController {
         pipelineStrategyDAO.getPipelinesByApplication(application)
     }
 
+    @RequestMapping(value = '{id:.+}/history', method = RequestMethod.GET)
+    Collection<Pipeline> getHistory(@PathVariable String id) {
+        try {
+            return pipelineStrategyDAO.getPipelineHistory(id)
+        } catch (UnsupportedOperationException ignored) {
+            return []
```

Comment:
if the operation is supported, is the current version one of the values returned?

if it is, should the unsupported case just return the current value, or should the history request filter out the current value?